src/index.html

```diff
@@ -22,6 +22,369 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </section>
+          <span class="reviews_count">Reviews: 5</span>
+        </div>
+        <div class="product-card__price">
+          <span class="product-card__price-label">Price:</span>
+          <span class="product-card__price-sum">$2,199</span>
+        </div>
+        <a
+          href="#"
+          class="product-card__button"
+        >
+          Buy
+        </a>
+      </div>
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__image">
+          <img
+            class="product-card__image-content"
+            src="images/imac.jpeg"
+            alt="iMac"
+          />
+        </div>
+        <p class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__reviews">
+          <section class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </section>
+          <span class="reviews_count">Reviews: 5</span>
+        </div>
+        <div class="product-card__price">
+          <span class="product-card__price-label">Price:</span>
+          <span class="product-card__price-sum">$2,199</span>
+        </div>
+        <a
+          data-qa="hover"
+          href="#"
+          class="product-card__button"
+        >
+          Buy
+        </a>
+      </div>
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__image">
+          <img
+            class="product-card__image-content"
+            src="images/imac.jpeg"
+            alt="iMac"
+          />
+        </div>
+        <p class="product-card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__reviews">
+          <section class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </section>
+          <span class="reviews_count">Reviews: 5</span>
+        </div>
+        <div class="product-card__price">
+          <span class="product-card__price-label">Price:</span>
+          <span class="product-card__price-sum">$2,199</span>
+        </div>
+        <a
+          data-qa="hover"
+          href="#"
+          class="product-card__button"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The product card elements are repeated multiple times with identical content. Consider using a loop or a template in a JavaScript framework to dynamically generate these cards to avoid repetition and improve maintainability.